### ISSUE TITLE
RecordRTC to Node.js problem

### ISSUE THREAD

**Issue by PiotrekFilipecki** [muaz-khan/RecordRTC#265]
Hello,  I have implemented RecordRTC to Node.js example in my node app with user authentication and I have problem like this:
![zrzut ekranu 2017-05-20 o 22 21 00](https://cloud.githubusercontent.com/assets/9655458/26279084/a77ad028-3daa-11e7-9c1c-01a11270ea90.png)

And now, I'm curious if its due to this that I have an mvc architecture, so my index file is in views folder which causes that fileName has wrong regexp implemented?

**Comment by muaz-khan** [comment]
Did you fix this?

**Comment by PiotrekFilipecki** [comment]
Yes I did. But now I am struggling with rewriting http post method to express. My app is supposed to record a 15 second fragments repeatedly. With use of your example I have developed a well working example but when I try to rewrite it to express it works like this: first fragment which is being recorded on load is also saved properly, unfortunately when app is supposed run event which records again, app crashes. Do you have any idea how to handle this problem?
